src/components/card/Card.tsx

```diff
@@ -1,22 +1,38 @@
-import BookBody from '../../model/Book'
+import { useEffect, useState } from 'react'
+
+import { Book } from '../../model/Book'
 import './Card.css'
+import placeholderImg from '../../assets/placeholder/placeholderForBook.png'
 
-interface Book {
-  book: BookBody
+interface Books {
+  book: Book
 }
 
-const Card = () => {
+const Card = ({ book } : Books) => {
+  const [ placeholder , setPlaceholder ] = useState('')
+
+  useEffect(() => {
+    setPlaceholder(placeholderImg)
+  },[])
   return (
     <div className='card_container'>
+      <img className='card_img' src={book.Cover ? `data: image/png;base64, ${book.Cover}` : placeholder} />
       <div className='card_title'>
-        <p>Title</p>
-      </div>
-      <div className='card_img'>
-        <img className='card_img' src='https://images.unsplash.com/photo-1536323760109-ca8c07450053' />
+        <p>{book.Title}</p>
       </div>
       <div className='card_titles'>
-        <p>Authors:</p>
-        <p>Details</p>
+        <p>{book.Description?.substring(0,30)}</p>
+      </div>
+      <div className='card_author'>
+        {book.Authors &&
+        book.Authors.map((authors) => (
```

Comment:
since map is iterating through array of authors, each element means one author.

Reply:
Yes i know. I changed name authors to author.